db/migrations/9.4.04.sql

```diff
@@ -0,0 +1,247 @@
+DO $$
+BEGIN
+   IF EXISTS (
+      SELECT 1
+      FROM information_schema.columns 
+      WHERE table_schema = 'public'
+            AND table_name = 'accounts'
+            AND column_name = 'identity_id'
+      ) 
+   THEN 
+      IF EXISTS (
+         SELECT 1
+         FROM information_schema.columns 
+         WHERE table_schema = 'public'
+               AND table_name = 'accounts'
+               AND column_name = 'identity_id'
+               AND is_nullable = 'NO' )
+      THEN
+         ALTER TABLE public.accounts ALTER COLUMN identity_id DROP NOT NULL;
+      END IF;
+   END IF;
+END $$;
+
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
We should avoid adding any new extensions. Idea is to have a db state which would be same if a user starts syncing from scratch without running migrations